release_creation/github.py

```diff
@@ -0,0 +1,35 @@
+import os
+from semantic_version import Version, match
+from github import Github
```

Comment:
Probably on your radar for the next piece regarding the GH workflow, but if not - a requirements.txt would be handy for local development + probably necessary for the GH workflow